.gitignore

```diff
@@ -1,3 +1,6 @@
 /internal/validation/testdata/graphql-js
 /internal/validation/testdata/node_modules
 /vendor
+.DS_Store
+.idea/
+.vscode/
```

Comment:
Unrelated change! Please, restore this file - these need to be in your global ignore file.

Reply:
I have restored this file